templates/compositions/ec-component-content-item-block/content-item-block.html.twig

```diff
@@ -0,0 +1,93 @@
+{% spaceless %}
+
+{#
+  /**
+   * @file
+   * Theme implementation for a ECL Content Item Block.
+   */
+  Parameters:
+    - "content_item_block" (associative array) (default: predefined structure): format:
+        {
+          type: '', (string) (default: 'one'): Type of Content Item Block
+          title: '', (string) (default: ''): Title of Content Item Block
+          items: [], (array) (default: []): Content Item's for the Content Item Block
+          button: {}, (associative array) (default: {}): Predefined structure compatible with Link component.
+        }
+    - "extra_classes" (string) (default: '')
+    - "extra_attributes" (array) (default: []): format: [
+        {
+          "name" (string) (default: ''),
+          "value" (string) (default: '')
+        },
+        ...
+      ]
+#}
+
+{# Internal properties #}
+
+{% set _content_item_block = {
+  type: 'one',
+  title: '',
+  items: [],
+  button: {}
+} %}
+
+{% if content_item_block is defined %}
```

Comment:
We don't need to do this in twig: ` is defined`, the code will be compiled in the right PHP code, let's remove them all, it clutters the code.

Reply:
:+1:

Reply:
Done :+1: